docs/modules/ROOT/pages/editing.adoc

```diff
@@ -48,9 +55,26 @@ for your user or run your shell/terminal as administrator to have this feature w
 You can also use `jbang edit --live` and `jbang` will launch your editor while watching
 for file changes and regenerate the temporary project to pick up changes in dependencies.
 
-== IDE and Editor support
+== IDE Support
+
+Some IDE's have dedicated JBang support and is recommended to use those if you can; by using `jbang edit .` (the dot is important) to edit a directory directly. 
+Wit this approach you avoid the symbolic links and temporary location and simply have the IDE understand the implicit project structure as understood by JBang.
+
+The following IDE's have dedicated support (at time of writing): 
+
+:sicons: https://simpleicons.org/icons
+.IDE's with JBang support
+[width=75%,frame=none,grid=none]
+|===
+|image:{sicons}/visualstudiocode.svg[50,50] https://code.visualstudio.com[Visual Studio Code]
+|image:{sicons}/eclipseide.svg[50,50] https://www.eclipse.org/downloads/download.php[Eclipse]
+|image:{sicons}/intellijidea.svg[50,50] https://www.jetbrains.com/idea/download[IntelliJ Idea]
+|image:{sicons}/gitpod.svg[50,50] https://www.gitpod.io[GitPod]
+|===
+
+== Generic edit support
 
-The `edit` feature been tested with the following IDE's:
+The generic `edit` feature using temporary directory and symbolic links been tested with the following IDE's:
```

Comment:
Same here: shouldn't we mention that's called sandbox mode and that you have to enable it explicitly using an option?

Reply:
reworded